wezterm-ssh/Cargo.toml

```
@@ -31,10 +31,11 @@ portable-pty = { version="0.8", path = "../pty" }
 regex = "1"
 smol = "1.2"
 ssh2 = {version="0.9.3", features=["openssl-on-win32"], optional = true}
-libssh-rs = {version="0.2.1", features=["vendored"], optional = true}
-#libssh-rs = {path="../../libssh-rs/libssh-rs", features=["vendored"], optional = true}
+# libssh-rs = {version="0.2.1", features=["vendored"], optional = true}
+libssh-rs = {path="../../libssh-rs/libssh-rs", features=["vendored"], optional = true}
```

Should be able to grab this from crates.io now!

```suggestion
libssh-rs = {version="0.3.1", features=["vendored"], optional = true
```

---

Done.